styles/csrt.css

```diff
@@ -0,0 +1,127 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+
+
+.cartPageNav {
```

Comment:
when you change the div to nav change this here to a nav ruleset without a class